src/components/Navigation/index.js

```diff
@@ -0,0 +1,43 @@
+import { AppBar, Hidden, Toolbar } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React from "react";
+
+import DesktopNavigation from "./DesktopNavigation";
+import MobileNavigation from "./MobileNavigation";
+
+const useStyles = makeStyles(({ palette, breakpoints, typography }) => ({
+  root: {
+    backgroundColor: palette.background.default,
+  },
+  section: {
+    width: "100%",
+    [breakpoints.up("lg")]: {
+      maxWidth: typography.pxToRem(1190),
+    },
```

Comment:
Why do we need this? We already have a Section component that has the exact size we need.
![Screenshot 2021-08-04 at 09-26-10 PesaYetu - web DEV](https://user-images.githubusercontent.com/1779590/128132332-a1083871-855f-46c4-b705-c60b0b62a0a1.png)